### DISCUSSION TITLE
Solution criteria description error in bedtools tutorial

### DISCUSSION THREAD

**Discussion by ChuanyiZ** [sandbox-bio/sandbox.bio#10] [Q&A] [answered]
The description of step 17 in the bedtools tutorial seems incorrect. It says:
> File `splice-sites.bed` contains a list of all regions of 500kb in the genome

Should it be "contains a list of regions of the splicing sites"?

**Comment by robertaboukhalil** [comment, marked as answer]
Thanks for catching that @ChuanyiZ! I'll make the update
